test/falco_tests.yaml

```diff
@@ -651,7 +651,7 @@ trace_files: !mux
 
   disabled_and_enabled_rules_1:
     exit_status: 1
-    stderr_contains: "Runtime error: You can not specify both disabled .-D/-T. and enabled .-t. rules. Exiting."
+    stderr_contains: "Runtime error: Could not initialize: You can not specify both disabled .-D/-T. and enabled .-t. rules. Exiting."
```

Comment:
Personally, I don't see a great reason to change the error output now with a prefix. This is not strictly relevant for this contribution, and also causes a user-facing change. I would advocate to leave the errors as is, and consider changing the format in the future.

Reply:
The reason for the "Could not initialize" is that there's an extra layer of functions now, specifically the application object and init. I can remove any prefix from that layer, and just pass lower level errors as-is.